### DISCUSSION TITLE
Bypass queue for specific dependency

### DISCUSSION THREAD

**Discussion by MaximilianRumpf** [renovatebot/renovate#29098] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

We have configured the prConcurrentLimit to its default, 10.
But for only one specific dependency, we would like to always create a PR, even if there are already 10 PRs for all other dependencies.
Can this be achieved somehow? E.g. as somehow marking this dependency as "security"-relevant which bypasses the queue?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
It's not supported, see this related feature request:

- https://github.com/renovatebot/renovate/issues/10546
